.vscode/settings.json

```
@@ -1,4 +1,5 @@
 {
+  "editor.defaultFormatter": "esbenp.preter-vscode",
```

esbenp.prettier-vscode

---

Oh god, no idea how that happened, haha 😅 
```suggestion
  "editor.defaultFormatter": "esbenp.prettier-vscode",
```